leptonai/cli/objectstore.py

```diff
@@ -0,0 +1,169 @@
+"""
+KV is a module that provides a way to manage kvs on a workspace.
+"""
+
+import click
+import os
+
+from rich.table import Table
+
+from .util import (
+    console,
+    click_group,
+    get_connection_or_die,
+    explain_response,
+    sizeof_fmt,
+)
+from leptonai.api import objectstore as api
+
+
+_max_upload_file_size_limit = 4995 * 1024 * 1024
+
+
+@click_group()
+def objectstore():
+    """
+    Manage the object store on the Lepton AI cloud. (beta)
+    """
+    pass
+
+
+@objectstore.command()
+@click.option("--key", "-k", help="Object key", type=str, required=True)
+@click.option("--file", "-f", help="File to write to.", type=str, default=None)
+@click.option("--bucket", "-b", help="Bucket name", type=str, default="private")
+@click.option(
+    "--return-url",
+    "-u",
+    help=(
+        "Return the url of the object instead of the content. If specified, --file"
+        " takes no effect."
+    ),
+    is_flag=True,
+)
+def get(key, file, bucket, return_url):
+    """
+    Gets the object with the given key.
+    """
+    if not file:
+        file = os.path.basename(key)
```

Comment:
shall we use key or just the basename of key?

Reply:
For convenience purposes I find myself always wanting to download and then put in the current folder. I guess user is recommended to actually explicitly specify file path.